orm/templates/varnish.vcl.j2

```diff
@@ -167,7 +167,11 @@ sub use_backend {
 {% if uses_sub_use_backend %}
 
 sub reconstruct_requrl {
+  if (variable.get("path") != "") {
   set req.url = variable.get("path");
+  } else {
+  set req.url = "/";
+  }
```

Comment:
Missing indentation inside the if/else clauses

Reply:
Done in https://github.com/SVT/orm/pull/46/commits/faa840a5c161385e882e7df12046a091dfb29704